docs/usage.rst

```diff
@@ -73,7 +73,7 @@ testing client.
     # Using Django's testing client
     @when(u'I visit "{url}"')
     def visit(context, url):
-        response = context.test.client.get(url)
+        context.response = context.test.client.get(url) # save response in context for next step
```

Comment:
Please put the comment (if really necessary) in a line above the code: Let's do no in-line comments if that can be avoided.

And when you're at it, can you please also modify lines 87 and 88 as follows? (`@then` and comment)
``` python
     @then(u'I should see "{text}"')
     def visit(context, text):
         # compare with response from ``when`` step
         context.test.assertContains(context.response, text)
```
Thank you for your help and understanding!